app/scripts/metamask-controller.js

```diff
@@ -1470,6 +1471,11 @@ export default class MetamaskController extends EventEmitter {
    * @param {MessageSender} sender - The sender of the messages on this stream
    */
   setupUntrustedCommunication (connectionStream, sender) {
+    const { usePhishDetect } = this.preferencesController.store.getState()
+    // Check if phishing detection is enabled
+    if (!usePhishDetect) {
+      return
```

Comment:
It looks like this would prevent establishing a connection between MetaMask and dapps - the stream is setup below on line 1491.

Perhaps this should be added to the condition below instead? e.g.
`if (usePhishDetect && this.phishingController.test(hostname)) {`

Reply:
Good catch, I missed that, will fix

Reply:
done